cmd/plasmacli/store/keystore.go

```diff
@@ -139,7 +138,8 @@ func DeleteAccount(name string) error {
 	return nil
 }
 
-// Update either the name of an account or the passphrase for an account
+// UpdateAccount updates either the name of an account or the passphrase for
+// an account.
```

Comment:
I don't see how this updates the passphrase.

Reply:
the else statement ` if err = ks.Update(acc, password, updatedPassword); err != nil {`